<?xml version="1.0" encoding="UTF-8" ?>
<launch>
    <arg name="port" default="/dev/ttyUSB0" />
    <arg name="auto_ask_flag" default="False" />
    <rosparam command="load" file="$(find xbee_communication)/config/robot_xbee.yaml" />

    <node name="xbeee_server" pkg="xbee_communication" type="xbee_server.py" output="screen" required="true">
        <param name="port" value="$(arg port)" />
        <param name="auto_ask_flag" value="$(arg auto_ask_flag)" />
    </node>


</launch>
